flask_resty/pagination.py

```diff
@@ -564,25 +564,61 @@ class RelayCursorPagination(CursorPaginationBase):
     https://facebook.github.io/relay/graphql/connections.htm.
     """
 
+    page_info_arg = "page_info"
+
+    def __init__(self, *args, default_include_page_info=False, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._default_include_page_info = default_include_page_info
+
+    def get_page_info(self, query, view, field_orderings, cursor):
+        include = flask.request.args.get(
```

Comment:
thought (non-blocking): would be nice to properly deserialize the string to bool with marshmallow if it's easy. nbd tho

Reply:
maybe

```python
    include = self.deserialize_value(fields.Boolean(), flask.request.args.get(....)
```